.github/workflows/test-build-deploy.yml

```diff
@@ -35,10 +40,15 @@ jobs:
   test:
     runs-on: ubuntu-20.04
     container:
-      image: quay.io/cortexproject/build-image:upgrade-to-go1.19.3-e8b98ddc0
+      image: quay.io/cortexproject/build-image:upgrade-to-go1.20.1-pr-5159
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v2
+      - name: Setup Git safe.directory
+        run: |
+          echo "this step is needed because when running in container, actions/checkout does not set sefe.directory effectively."
```

Comment:
```suggestion
          echo "this step is needed because when running in container, actions/checkout does not set safe.directory effectively."
```

Reply:
sorry about the copy and paste error ...